Code_parser.cpp

```diff
@@ -0,0 +1,54 @@
+#include "Code_parser.h"
+
+void Code_parser::doParse(const string file_path) {
+
+	ifstream read_file(file_path);
```

Comment:
I think it is better to verify if file exists first.

Reply:
Resolved